shell/platform/fuchsia/flutter/text_delegate.h

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Text editing functionality delegated from |PlatformView|.
+// See |TextDelegate| for details.
+
+#ifndef FLUTTER_SHELL_PLATFORM_FUCHSIA_TEXT_DELEGATE_H_
+#define FLUTTER_SHELL_PLATFORM_FUCHSIA_TEXT_DELEGATE_H_
+
+#include <memory>
+
+#include <fuchsia/ui/input/cpp/fidl.h>
+#include <fuchsia/ui/input3/cpp/fidl.h>
+#include <fuchsia/ui/views/cpp/fidl.h>
+#include <lib/fidl/cpp/binding.h>
+
+#include "flutter/lib/ui/window/platform_message.h"
+#include "flutter/shell/common/platform_view.h"
+#include "flutter/shell/platform/fuchsia/flutter/keyboard.h"
+
+#include "logging.h"
+
+namespace flutter_runner {
+
+// The channel name used for text editing platofrm messages.
+constexpr char kTextInputChannel[] = "flutter/textinput";
+
+// The channel name used for key event platform messages.
+constexpr char kKeyEventChannel[] = "flutter/keyevent";
+
+// FL(77): Terminate engine if Fuchsia system FIDL connections have error.
```

Comment:
Thanks for prodding me to check. I originally thought it would be hard to know, but it turns out that it was knowable. 

`FL-77` refers to the bug's old JIRA ID. I was able to find it at: https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=8668, so it's now referring to it.